app/src/main/java/com/github/swent/echo/ui/navigation/Navigation.kt

```diff
@@ -0,0 +1,60 @@
+package com.github.swent.echo.ui.navigation
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+
+/** The routes to the screens of the app */
+enum class Routes {
+    SIGN_IN,
+    MAP,
+}
+
+/**
+ * This class reprensent the different navigation actions
+ *
+ * @param navController the navigation controller
+ */
+class navigationActions(val navController: NavHostController) {
```

Comment:
Nitpicking: The convention is to start class names with capital letters: `NavigationActions`. :)

Reply:
done